database/open_source/projects.json

```
@@ -152,6 +152,13 @@
     "url": "https://github.com/TheTorProject",
     "category": "open-source",
     "subcategory": "projects"
+  },
+  {
+    "name": "Free Programming Books",
+    "description": "Freely available programming books.",
```

[suggestion] This description could use a bit more detail. Consider mentioning the purpose of this project and why people should contribute to it. That way, people would become more interested in contributing.
```suggestion
    "description": "Freely available programming books.",
```